hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/hive/HoodieCombineHiveInputFormat.java

```diff
@@ -392,7 +393,9 @@ public InputSplit[] getSplits(JobConf job, int numSplits) throws IOException {
       Arrays.stream(paths).forEach(path -> {
         final HoodieStorage storage;
         try {
-          storage = new HoodieHadoopStorage(path.getFileSystem(job));
+          FileSystem fs = path.getFileSystem(job);
+          storage = HoodieStorageUtils.getStorage(
+              HadoopFSUtils.getStorageConf(fs.getConf()), new Class<?>[] {FileSystem.class}, fs);
```

Comment:
Same here, we should use path and conf to construct storage instance, not FileSystem